packages/labs/ssr-react/src/test/object-test-element.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {html, LitElement} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+
+@customElement('object-test-element')
+export class ObjectTestElement extends LitElement {
+  @property({type: Object})
+  user = {name: 'Somebody'};
+
+  override render() {
+    return html`<p>Hello, ${this.user.name}!</p>`;
+  }
+}
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'object-test-element': ObjectTestElement;
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace JSX {
```

Comment:
No action - just curious. What is this for? Is this to add a type for the jsx element?

Reply:
Yes, it allows usage of the bare lower case custom element tag in JSX without TS errors. Normally only built in HTML elements are allowed to be lower case.